internal/components/inspektorgadget/const.go

```diff
@@ -43,6 +43,7 @@ const (
 	inspektorGadgetChartRelease   = "gadget"
 	inspektorGadgetChartNamespace = "gadget"
 	inspektorGadgetChartURL       = "oci://ghcr.io/inspektor-gadget/inspektor-gadget/charts/gadget"
```

Comment:
Is there a version tag in the chart? If so, is it possible to always use the latest version when fetching the chart?

Reply:
Got it, that make sense!